src/components/TheHeader.vue

```diff
@@ -1,9 +1,11 @@
 <template>
   <header>
     <center-wrapper :top="true">
-      <div class="leadin">A video archive of</div>
-      <h1>Police Abuses</h1>
-      <div class="leadout">{{ total }} Videos</div>
+      <router-link to="/">
```

Comment:
I can't approve this, this is not accessible :). You can't place a div in a link.

It should be `<h1><router-link>Police Abuses</router-link></h1>`

Reply:
Since HTML5 [anchor elements can contain noninteractive flow content](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Properties), so it's ok now.

Reply:
Technically you are correct  😄 . However, a screenreader can't make distinguish the elements, and reads it aloud as a single line of text. That's why i use that weird clickevent on the `IncidentListItem`. That trick is from [this website](https://inclusive-components.design/cards/).

Reply:
To fix this, i think it's fine to make only the `h1` clickable:
`<h1><router-link>Police Abuses</router-link></h1>`

Reply:
Ok, changed it to your suggestion.